lighthouse-core/test/runner-test.js

```diff
@@ -522,14 +520,10 @@ describe('Runner', () => {
 
     return Runner.run({}, {url, config}).then(results => {
       assert.deepEqual(results.artifacts.ViewportDimensions, ViewportDimensions);
-
-      for (const method of Object.keys(computedArtifacts)) {
```

Comment:
artifacts from `Runner.run()` no longer include computed artifact methods on them. This and below just removes the check that they are on the returned artifacts object